### PR TITLE
quincy: cephadm: change ceph-ci image from master to quincy

### PR DESCRIPTION
**Pull request by nizamial09**
quincy image is available in the quay.io repo and we should use it for
the quincy branch for now atleast until [v17 is released.](https://github.com/ceph/ceph/pull/45196)

Signed-off-by: Nizamudeen A <nia@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by adk3798 — APPROVED** [review]

**Comment by nizamial09** [comment]
jenkins test dashboard cephadm

File: src/cephadm/cephadm [hunk 1]
```diff
@@ -45,7 +45,7 @@ from pathlib import Path
 FuncT = TypeVar('FuncT', bound=Callable)
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:master'
+DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:quincy'
```
Comment (epuertat) [inline src/cephadm/cephadm:48]: It seems that in pacific we're using `v16` tag: https://github.com/ceph/ceph/blob/pacific/src/cephadm/cephadm#L51-L51

**Review by epuertat — APPROVED** [review]

**Review by mgfritch — APPROVED** [review]
this is exactly what we did for octopus and pacfic, but for completeness wondering if we should add a similar change in the mgr:
https://github.com/ceph/ceph/blob/8299cd4c9bdb8ebccc5648906737e154090a2499/src/pybind/mgr/cephadm/module.py#L93

**Comment by nizamial09** [comment]
> wondering if we should add a similar change in the mgr

@mgfritch I am not sure how that change would go. Besides, I am not seeing any reference to `master` tag being used there.

**Comment by ljflores** [comment]
http://pulpito.front.sepia.ceph.com/yuriw-2022-02-28_21:23:00-rados-wip-yuri-testing-2022-02-28-0823-quincy-distro-default-smithi/

Failures, unrelated:
    https://tracker.ceph.com/issues/54029
    https://tracker.ceph.com/issues/54439
    https://tracker.ceph.com/issues/50280

Details:
    1. orch:cephadm/workunits/{agent/on mon_election/connectivity task/test_orch_cli} test failing - Ceph - Orchestrator
    2. LibRadosWatchNotify.WatchNotify2Multi fails - Ceph - RADOS
    3. cephadm: RuntimeError: uid/gid not found - Ceph

**Pull request merged**